File: reverse_text_string_word.py
text = 'Hello how are you?'
def reverse_text_string_word(text):
  end = -1
  reversed_text = ''
  for i in range(-1,-(len(text)+1),-1):
    if(text[i] == ' '):
      for j in range(i+1,end+1, 1):
        reversed_text += text[j]
      reversed_text += ' '
      end = i - 1
  if end != -1:
    for j in range(0,(len(text)+end)+1,1):
      print(text[j])
      reversed_text += text[j]
  return reversed_text

print(reverse_text_string_word(text))

File: reverse_text_string_word.js
function reverse_text_string_word(text){
    let end = text.length;
    let reversed_text = '';
    
    for (let i=text.length-1; i >= 0; i--){
        if (text[i] == ' '){
            for (let j=i+1; j<end; j++){
                reversed_text += text[j];
            }
            reversed_text += ' ';
            end = i;
        }
    }

    if (end!= text.length){
        for (let j=0; j<end; j++){
            console.log(reversed_text);

            reversed_text += text[j];
        }
    }
    return reversed_text;
}
console.log(reverse_text_string_word('Who are you fahim'));